maxim/models/test_run.py

```
@@ -19,26 +19,38 @@ class YieldedOutputTokenUsage:
     total_tokens: int
     latency: Optional[float]
 
+    def __init__(
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)